security/s2a/internal/crypter/counter.go

```diff
@@ -0,0 +1,37 @@
+package crypter
+
+import (
+	"errors"
+)
+
+// counterLen is the byte length of the counter.
+const counterLen = 8
```

Comment:
I don't think we need this anymore?

Reply:
Ack. I've removed it for now since it isn't used, but I suspect we'll have to add it back later when implementing the half connection because we need to know how many bytes the counter's uint64 represents.